src/nvim/ex_docmd.c

```diff
@@ -7858,19 +7858,21 @@ static void ex_stopinsert(exarg_T *eap)
  */
 void exec_normal_cmd(char_u *cmd, int remap, bool silent)
 {
+  // Stuff the argument into the typeahead buffer.
+  ins_typebuf(cmd, remap, 0, true, silent);
+  exec_normal(false);
+}
+
+void exec_normal(int was_typed)
```

Comment:
Updated. Thanks. I also changed the parameter from `int` to `bool` as that seemed better. I honestly wasn't sure how to describe the `was_typed` parameter as the `exec_normal` method is only called once and the argument is always true so I just wrote something very generic.